src/settings/HRIDHandling/HRIDHandlingForm.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { Router } from 'react-router';
+import { createMemoryHistory } from 'history';
+import '../../../test/jest/__mock__';
+import { renderWithIntl, translationsProperties } from '../../../test/jest/helpers';
+import HRIDHandlingForm from './HRIDHandlingForm';
+
+const history = createMemoryHistory();
+const defaultProps = {
+  initialValues: {
+    metadata: {}
+  },
+  handleSubmit: jest.fn(),
+  pristine: true,
+  submitting: true,
+  form: {
+    reset: jest.fn()
+  },
+  render: jest.fn()
+};
+
+const renderHRIDHandlingForm = (props) => renderWithIntl(
+  <Router history={history}>
+    <HRIDHandlingForm {...props} onSubmit={jest.fn()} />
+  </Router>,
+  translationsProperties
+);
+
+describe('HRIDHandlingForm', () => {
+  it('Component should render', () => {
+    renderHRIDHandlingForm(defaultProps);
+    expect(screen.getAllByText('HRID handling').length).toBe(2);
+    expect(screen.getByText('ViewMetaData')).toBeInTheDocument();
+    expect(screen.getByText('After initial data migration, new FOLIO HRIDs are assigned sequentially, based on the starting number in these settings')).toBeInTheDocument();
```

Comment:
Can you reference by translation ID instead in case the text is updated by translators? See https://github.com/folio-org/ui-organizations/blob/d956a3d71805cb9f337e691f7ce06431856f6f42/src/Organizations/OrganizationDetails/OrganizationVendorInfo/OrganizationVendorInfo.test.js#L27 for example.

Reply:
Hi @ryandberger ,
We could not find translation ID, so instead used className (line 95) as of mentioned in js file.

![image](https://user-images.githubusercontent.com/100114810/227420509-65a162e9-f4a4-4e88-8611-5d80d3d1312e.png)